src/styles/util.scss

```diff
@@ -89,3 +89,14 @@
 .u-color-text--muted {
   color: var.$color-text--light;
 }
+
+// TODO: This is a temporary utility class to allow elements in the sidebar
```

Comment:
I think the thing I mentioned before that I wasn't clear about is _which_ elements does this refer to. Perhaps this might be easier to do by mentioning some examples. For example, the first sentence here could be:

"Utility class to space top-level containers (panels, the thread list etc.) in the sidebar".

The rest of the comment is fine.